ts/features/payments/common/utils/index.ts

```diff
@@ -29,20 +29,36 @@ export const TRANSACTION_LOGO_CDN = `${contentRepoUrl}/logos/organizations`;
  * based on the transaction status.
  */
 
-export const getBadgeTextByTransactionStatus = (
-  transactionStatus: ListItemTransactionStatusWithBadge
-) => {
+export const getBadgeTextAndVariantByTransactionStatus = (
```

Comment:
Just a naming suggestion:
```suggestion
export const getBadgeByTransactionStatus = (
```
or 
```suggestion
export const getBadgePropsByTransactionStatus = (
```